.github/workflows/pre-merge.yaml

```diff
@@ -14,7 +14,7 @@ on:
   push:
     branches:
       - "release/**"
-      - "feature/**"
+      #- "feature/**" # Run `promote` workflow instead.
```

Comment:
No, to me you want to remove the previous one `"release/**"`

Reply:
Fixed!